app/controllers/api/v1/albums_controller.rb

```diff
@@ -6,7 +6,9 @@ def index
   end
 
   def show
-    render json: Album.find(params[:id])
+    album = Album.find(params[:id])
+    reviews = album.reviews
+    render json: album
```

Comment:
here, you are finding the album to show _and_ creating a variable for the reviews associated with that album, but not rendering `reviews`. Line 10 can be removed since you can call `album.reviews` to get all the reviews in your component